packages/devtools/lib/src/ui/theme.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'fake_flutter/fake_flutter.dart';
+import 'flutter_html_shim.dart';
+
+bool _isDarkTheme = false;
+bool get isDarkTheme => _isDarkTheme;
```

Comment:
nit: reverse order of the getter and the var?

Reply:
done. Also added a comment.